src/SFA.DAS.LevyTransferMatching.Api/Models/ApplicationRejectRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace SFA.DAS.LevyTransferMatching.Api.Models
+{
+    public class ApplicationRejectRequest
```

Comment:
Why not `RejectApplicationsRequest`?

Reply:
Yes sure. I have refactored it and renames it now.